climakitae/uncertain_utils.py

```diff
@@ -239,7 +311,9 @@ def grab_temp_data(copt):
     Attributes
     ----------
```

Comment:
This should be "Parameters" not "Attributes"

Reply:
For some reason this has a weirdly large effect on how they are displayed in the readthedocs page haha

Reply:
got it, thanks!